### ISSUE TITLE
Can not play on Android device browser

### ISSUE THREAD

**Issue by mark5655** [googleads/videojs-ima#197]
First thanks for this wonderful project! 
Bug: On Android device browser you can't click player to start playing. Works OK on desktop. Any idea how to fix this?
Thanks!

**Comment by shawnbuso** [comment]
Is the iPhone sample working for you on Android? We currently have an issue with the non-iPhone samples on Android. You can see more details here: https://github.com/googleads/videojs-ima/issues/142

**Comment by mark5655** [comment]
Yes, that one is working but sadly not really what i needed. But keep on the good work! Other players charge a lot for what you offer here for free :)

**Comment by shawnbuso** [comment]
Glad you like the plugin! Is there anything in particular that you're looking for in the iPhone sample that is provided in the other examples? In theory it should be able to do anything those other samples do.

**Comment by mark5655** [comment]
:) Yes, it needs player controls (pause, time, video forward/backward... standard videojs) and the way to add url to video file as embedded code, now video url is in ads.js which is not easy for me to use when i use same template for all videos on my website. My programming skills are not on very high level.

**Comment by shawnbuso** [comment]
Ah ok - hopefully #142  will be unblocked soon and we'll be able to resume using the other samples on Android.

**Comment by shawnbuso** [comment]
Closing as a duplicate of #142
